build/source/netcdf/read_icond.f90

```diff
@@ -292,10 +294,9 @@ subroutine read_icond(iconFile,                      & ! intent(in):    name of
     ! get the index in the file: single HRU
     if(restartFileType==singleHRU)then
      ixFile = 1  ! use for single HRU restart file
-
     ! get the index in the file: multi HRU
     else
-     ixFile = iHRU_local
+     ixFile = startGRU + iHRU_local - 1
```

Comment:
Why the `-1` offset relative to `startGRU`? That seems a C things (if `startGRU` starts at ` and indices start at 0). Is this correct? I guess if both `startGRU` and `iHRU_local` are 1 this works.

Reply:
right.  I think this is the equivalent of using ixFile = iHRU_global, incidentally.  using iHRU_local was the bug before.

Reply:
Exactly, basically it's just accounting for 2 things that both start at 1